test/.eslintrc.json

```diff
@@ -9,5 +9,16 @@
 	},
 	"rules": {
 		"arrow-body-style": "off"
+	},
+	"settings": {
+		"import/resolver": {
+			"node": {
```

Comment:
Is `node` necessary?

Reply:
I think so. It seems like required key for the plugin
[https://www.npmjs.com/package/eslint-import-resolver-node](https://www.npmjs.com/package/eslint-import-resolver-node)